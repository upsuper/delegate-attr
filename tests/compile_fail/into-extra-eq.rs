use delegate_attr::delegate;

struct Inner;

impl Inner {
    fn answer(&self) -> u32 {
        42
    }
}

struct Wrapper(Inner);

#[delegate(self.0)]
impl Wrapper {
    #[into = "a"]
    fn answer(&self) -> u64 {}
}

fn main() {}
