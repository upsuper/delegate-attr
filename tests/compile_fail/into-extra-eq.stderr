error: unexpected argument
  --> $DIR/into-extra-eq.rs:15:12
   |
15 |     #[into = "a"]
   |            ^
